db/dcrsqlite/retrieveblockfeeinfo_test.go

```diff
@@ -0,0 +1,23 @@
+package dcrsqlite
+
+import (
+	"testing"
+
+	"github.com/decred/dcrdata/testutil"
+)
+
+func TestEmptyDBRetrieveBlockFeeInfo(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := InitTestDB(DBPathForTest())
+	testEmptyDBRetrieveBlockFeeInfo(db)
```

Comment:
Is it necessary to have `InitTestDB` run repeatedly.  It would be much more concise to have lots of these "empty" tests just use the one empty db you init.  This is one reason I referred you to golang's established test harnesss setup: https://golang.org/pkg/testing/#hdr-Main.

Reply:
Thank you.  I'll look into it.

Reply:
I found the following solution to address the issue: 
```
var ReusableEmptyDB *DB = nil

// ObtainReusableEmptyDB returns a single reusable instance of an empty DB. The instance
// is created once during the first call. All the subsequent calls will return
// result cached in the ReusableEmptyDB variable above.
func ObtainReusableEmptyDB() *DB {
	if ReusableEmptyDB == nil {
		testName := "ReusableEmptyDB"
		testutil.ResetTempFolder(&testName)
		target := filepath.Join(testName, testutil.DefaultDBFileName)
		targetDBFile := testutil.FilePathInsideTempDir(target)
		ReusableEmptyDB = InitTestDB(targetDBFile)
	}
	return ReusableEmptyDB
}
```